tiles/src/main/java/com/protomaps/basemap/layers/Pois.java

```diff
@@ -355,7 +356,9 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
             }
           }
         } else if (kind.equals("cemetery") ||
-          kind.equals("school")) {
+          kind.equals("school") ||
+          kind.equals("allotments") ||
```

Comment:
Allotment POIs should appear later than schools, more like zoom 15 or 16

(see also how Tilezen sets the value: https://github.com/tilezen/vector-datasource/blob/master/yaml/pois.yaml#L1258-L1266)

Reply:
Then I'll add it such that allotments larger than 250m² get shown from zoom level 15, and smaller ones from 16 (basically last part of this branch).